criu/page-xfer.c

```diff
@@ -386,14 +384,18 @@ static int open_page_local_xfer(struct page_xfer *xfer, int fd_type, unsigned lo
 		if (opts.stream)
 			goto out;
 
-		pfd = openat(get_service_fd(IMG_FD_OFF), CR_PARENT_LINK, O_RDONLY);
-		if (pfd < 0 && errno == ENOENT)
-			goto out;
+		pfd = open_parent(get_service_fd(IMG_FD_OFF));
+		if (pfd < 0) {
+			if (pfd == -1)
+				goto out;
+			else
+				goto err_cli;
+		}
```

Comment:
I think it would have been nicer to do something like
```
if (open_parent(get_service_fd(IMG_FD_OFF), &pfd))
	goto err_cli;
if (pfd < 0)
	goto out;
```
rather than use `-1` and `-2` as the return values of `open_parent()`.